pkgs/applications/misc/phoc/default.nix

```diff
@@ -0,0 +1,66 @@
+{ stdenv
+, fetchFromGitLab
+, meson
+, ninja
+, pkgconfig
+, wrapGAppsHook
+, libinput
+, gnome3
+, glib
+, gtk3
+, wayland
+, dbus
+, cmake
+, libdrm
+, libxkbcommon
+, wlroots
+}:
+
+stdenv.mkDerivation rec {
+  pname = "phoc";
+  version = "0.1.7";
+
+  src = fetchFromGitLab {
+    domain = "source.puri.sm";
+    owner = "Librem5";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1007kxap87h1bi79sz45lfb478d93pq0676cr5rnhbanljm1n28n";
+  };
+
+  nativeBuildInputs = [
+    meson
+    ninja
+    pkgconfig
```

Comment:
In new expressions, `pkg-config` should be used instead of the `pkgconfig` alias.